expression/builtin_info.go

```diff
@@ -810,6 +810,11 @@ func (b *builtinTiDBDecodeSQLDigestsSig) Clone() builtinFunc {
 }
 
 func (b *builtinTiDBDecodeSQLDigestsSig) evalString(row chunk.Row) (string, bool, error) {
+	pm := privilege.GetPrivilegeManager(b.ctx)
+	if pm != nil && !pm.RequestVerification(b.ctx.GetSessionVars().ActiveRoles, "", "", "", mysql.ProcessPriv) {
+		return "", true, errSpecificAccessDenied.GenWithStackByArgs("PROCESS")
+	}
+
```

Comment:
maybe we can move this check to  `tidbDecodeSQLDigestsFunctionClass#getFunction`~?

because it seems doesn't related to `row`, move to `tidbDecodeSQLDigestsFunctionClass` can check 1 time for code snippets like this:

```
create table tt(d varchar(100));
insert into tt values('["aa"]'),('["aa"]'),('["aa"]');
select tidb_decode_sql_digests(d) from tt;
```